### ISSUE TITLE
Train over new data

### ISSUE THREAD

**Issue by sicotronic** [openai/generating-reviews-discovering-sentiment#30]
I'm sorry if it is just me that couldn't find it, but can you also provide the code to retrain the model on new data?

**Comment by friesel** [comment]
Agree: https://github.com/openai/generating-reviews-discovering-sentiment/issues/17

**Comment by pickou** [comment]
Wish to get the reply from the author as soon as possible. I wonder how to fine tune the model using my own dataset.

**Comment by loretoparisi** [comment]
👍

**Comment by jonnykira** [comment]
Hello,

Here is code to train a multiplicative LSTM language model in Tensorflow:

[https://github.com/jonnykira/Tensorflow_mLSTM](url)

Hope it works! please feel free to leave feedback!

**Comment by nkooli** [comment]
i am trying to test over a new dataset. I trained the mlstm over my dataset using the code in "multiplicative LSTM language model in Tensorflow". This generates me a three files : model.data, model.index, model.meta 
How can generates the 15 .npy files (0.npy, 1.npy, ..., 14.npy) to test the sentiment analysis code in https://github.com/openai/generating-reviews-discovering-sentiment
thank you !

**Comment by nkooli** [comment]
see the response in [https://github.com/jonnykira/Tensorflow_mLSTM/issues/1](url)
